x-pack/plugins/ml/server/models/data_recognizer/modules/network/ml/datafeed_high_count_by_destination_country.json

```diff
@@ -0,0 +1,25 @@
+{    "job_id": "high-count-by-destination-country",
+   "indices": ["logs-*","filebeat-*","packetbeat-*"],
+    "max_empty_searches": 10,
+      "query": {
+      "bool": {
```

Comment:
@randomuserid you should use terms filters over match_phrase filters, as they're more performant. You also have a dangling `must`, `should`, and an unneeded `match_all`. This bool query can be rewritten:

```
"bool": {
    "filter": [
        {"term": {"event.category": "network"}},
        {"exists": {"field": "destination.geo.country_name"}}
    ]
}
```

Reply:
done